config/config_as_cty_test.go

```diff
@@ -44,6 +44,14 @@ func TestTerragruntConfigAsCtyDrift(t *testing.T) {
 					Execute:  []string{"true"},
 				},
 			},
+			ErrorHooks: []ErrorHook{
+				ErrorHook{
+					Name:     "init",
+					Commands: []string{"init"},
+					Execute:  []string{"true"},
+					OnErrors: []string{".*"},
+				},
+			},
```

Comment:
Why is the ErrorHooks block not included under the Terraform one above, so that it would be nested at the same level as the Before/AfterHooks?

Reply:
I didn't understand the comment, it is at same level as before/after hooks from what I can see

Reply:
@smitthakkar96 Did you make this change? Just wanted to give this thread a bump, since it might have gotten missed!

Reply:
Hey sorry I didn't get the comment, what am I missing here?

Reply:
Ah, I see. I was seeing things indented incorrectly due to the GitHub UI. Looking at the raw file, everything is formatted right. Closing this thread. Sorry for the noise!